pkg/index/inverted/query.go

```diff
@@ -634,8 +634,29 @@ func (t *timeRangeNode) String() string {
 	return convert.JSONToString(t)
 }
 
+// PropertyQuery support query property with order.
+type PropertyQuery struct {
+	Query      index.Query
+	Order      *index.OrderBy
+	orderField string
+}
```

Comment:
```suggestion
```

The query should not include fields relevant to ordering. Internally, Bluge's sorting system operates as a separate data structure that is not accessible through the query tree. For this reason, sorting is not supported within the query itself. Therefore, the sorting operation should only be handled in the `property` package.